Src/DependencyCollector/Net46.Tests/DependencyTrackingTelemetryModuleTestNet46.cs

```diff
@@ -270,7 +342,7 @@ public void OnBeginOnEndAreNotCalledForAppInsightsUrl()
         private void ValidateTelemetryForDiagnosticSource(DependencyTelemetry item, Uri url, WebRequest request, bool success, string resultCode)
         {
             Assert.AreEqual(url, item.Data);
-            Assert.AreEqual(url.Host, item.Target);
+            Assert.AreEqual($"{url.Host}:{url.Port}", item.Target);
```

Comment:
Add a comment what we collect port only if its non-standard port like 80,443, and for local server the port is non-standard.
If possible, make a test localhost:80 or 443, and validate that we don't collect port in those cases.